canonical-websites/snapcraft.io
# If no_redirect set on logout, don't logout of bsi and go straight to snapcraft.io homepage

# Done

Fixes https://github.com/canonical-websites/snapcraft.io/issues/970

Also relies on https://github.com/canonical-websites/build.snapcraft.io/pull/1129

- Added query parameter to `/logout` to prevent a redirect loop

# QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/login and login
- Visit http://0.0.0.0:8004/logout?no_redirect=true
- See the snapcraft.io homepage
- Visit http://0.0.0.0:8004/login and login
- Visit http://0.0.0.0:8004/logout
- See the build.snapcraft.io homepage (https://github.com/canonical-websites/build.snapcraft.io/pull/1129) will redirect back to the snapcraft.io homepage
